about.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>WebDev Summit/about</title>
+    <link rel="stylesheet" href="./assets/css/about.css" />
+  </head>
+  <body>
+    <!-- header section -->
+    <header>
+      <nav>
+        <div class="mobile-menu">
+                    <button class="mobile-menu-button">&#9776;</button>
+                  </div>
+                  <div class="mobile-menu-options">
+                    <button class="mobile-menu-closing-btn">
+                        &times;
+                    </button>
+                    <ul class="mobile-menu-list-ul">
+                        <li class="mobile-menu-list-li"><a href="./index.html">Home</a></li>
+                        <li class="mobile-menu-list-li"><a href="./about.html">About</a></li>
+                    </ul>
+                  </div>
+        <div class="logo">
+          <a href="/Capstone-Project/index.html"
+            ><img src="./assets/imgs/logo.png" alt="Logo"
+          /></a>
+        </div>
+        <div class="menu">
+          <ul class="menu-list-ul">
+            <li class="menu-list-li"><a href="./index.html">Home</a></li>
+            <li class="menu-list-li"><a href="./about.html">About</a></li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <!-- main section -->
+    <main>
+      <section class="hero-section">
+        <h2>
+          Web Dev Creative <br />
+          Summit 2023
+        </h2>
+        <span class="hero-section-details">
+          Lorem ipsum dolor sit amet consectetur, adipisicing elit. Laudantium
+          soluta a incidunt ut quaerat, eos, nostrum ad omnis consequatur
+          quidem, dolores quod suscipit explicabo eaque laboriosam! Possimus
+          vero sapiente necessitatibus ut. Doloremque dolore nam, eveniet
+          quisquam vero dolores? Debitis eveniet inventore sapiente dignissimos
+          rem excepturi expedita similique. Magni, neque consequatur.
+        </span>
+        <span class="hero-section-contact us"
+          >Please contact us at abc@gmail.com</span
+        >
+      </section>
+      <section class="logo-explaining">
+        <h3>Web Dev Creative Summit 2023</h3>
+        <span class="logo-explaining-details">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Assumenda
+          voluptatum eos repudiandae fugit a earum adipisci vitae ratione
+          inventore eligendi ducimus magni vel praesentium reprehenderit facere
+          excepturi, quisquam similique voluptas! Cumque, obcaecati beatae?
+        </span>
+        <img src="./assets/imgs/logo.png" alt="logo" />
+      </section>
+      <section class="see-past-projects">
+        <h3>See the past web dev Global Summits</h3>
```

Comment:
- [ ] There is a divider missing under this heading. Please add it in order to match the Figma. 👍🏾